misc/community-marketing-kit.md

```diff
@@ -57,9 +57,9 @@ Here are a few channels to spread the words of the new Firefox launch. The follo
 * Organization: team up with other tech non-profit or open source organizations, schools, universities, and libraries to discuss web or tech literacy; bulletin board.
 * Mozilla: [Mozillagram](https://www.instagram.com/mozillagram/) will share photos from events. Below is the process to get your photos published:
 
-* Email request a week in advance to [Sierra Read](sreed@mozilla.com), in charge of social media. Once the community lead is granted access, the person or the group is allowed to take over the account for a few days at a time. During this period, they have direct access and are responsible for posting.
+  * Email request a week in advance to [Sierra Read](mailto:sreed@mozilla.com), in charge of social media. Once the community lead is granted access, the person or the group is allowed to take over the account for a few days at a time. During this period, they have direct access and are responsible for posting.
```

Comment:
mini-nit (flod taught me well ;) ): seems like there's a space there, that shouldn't be there. Otherwise looks fine!

Reply:
@Delphine, I hope I can retain all that flod taught me, :D. Are you suggesting the "extra" space at the beginning of * Email? This paragraph needs to be indented.  That's one of the corrections.

Reply:
@flodolo maybe your eagle eye can identify the extra space.

Reply:
Well, they're 2 spaces :-)

Ideally they should be 4 to be indented.